### ISSUE TITLE
makeotf: CMap file cannot support non-ASCII comment

### ISSUE THREAD

**Issue by LiXiangchen** [adobe-type-tools/afdko#147]
For instance:

### ↓ no problem

**% 测试文本**
1 begincidchar
<00004e00> 1001
endcidchar

### ↓ also no problem

1 begincidchar
**% Test text 1**
<00004e00> 1001 **% Test text 2**
endcidchar

### ↓ makeotfexe [FATAL] expecting hex string

1 begincidchar
**% 测试文本**
<00004e00> 1001
endcidchar

### ↓ makeotfexe [FATAL] expecting endcidcharrange; range count too small?

1 begincidchar
<00004e00> 1001 **% 测试文本**
endcidchar

**Comment by kenlunde** [comment]
I agree that comments, regardless of their content (ASCII versus non-ASCII), should be ignored.

**Comment by miguelsousa** [comment]
@kenlunde can you provide a simple set of files and commands to test this?

**Comment by kenlunde** [comment]
I just tried to reproduce this, and cannot. Perhaps @readroberts already fixed it?
